apps/docs/src/app/core/component-docs/popover/examples/popover-mobile/popover-mobile-example.component.scss

```diff
@@ -0,0 +1,7 @@
+.form-container div {
+    margin-bottom: 1rem;
+}
+
+.footer-buttons-container button {
+    margin-left: 1rem;
+}
```

Comment:
You can't have this styling here. We are not allowed to target html elements like `div` and `button`. This may lead to style leaking.

Reply:
Also, this styling will not be available to the user

Reply:
@InnaAtanasova do you recommend to remove this styling or move it somewhere else?
I think once it is for temporary content, it's okay if I use `style` attribute in html

Reply:
Removed, styles